README.md

```diff
@@ -132,7 +137,6 @@ Example commands:
 
 - [ ] **[Desktop verson]**
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe on _line 194_ is broken, kindly endeavor to fix it, and remove the guide note below it.